cms/templates/maintenance/_announcement_index.html

```diff
@@ -0,0 +1,49 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="studio-view maintenance-form">
+    <div class="form-create">
+        <div class="announcement-container">
+            % for announcement in announcement_list:
+                <div class="announcement-item">
+                    <div class="announcement-content">
+                        ${announcement.content | n} ## xss-lint: disable=mako-invalid-html-filter
+                    </div>
+
+                    <div class="actions-list">
+                        <a class="action-item announcement-edit"
+                          href="${ reverse('maintenance:announcement_edit', kwargs={'pk': announcement.pk}) }">
+                            <button class="btn-default">Edit</button>
+                        </a>
+                        <a class="action-item announcement-delete"
+                          href="${ reverse('maintenance:announcement_delete', kwargs={'pk': announcement.pk}) }">
+                            <button class="btn-default">Delete</button>
+                        </a>
+                    </div>
+                </div>
+            % endfor
+        </div>
+        <div class="actions">
+            <a href="${ reverse('maintenance:announcement_create') }">
+                <button class="action action-primary">${_('Create New')}</button>
+            </a>
+            % if is_paginated:
```

Comment:
Pagination doesn't seem to be working in the CMS view?  I've created 11 announcements, and they're all showing up.

Reply:
Ah whoops, fixed that one. I guess I unintentionally checked in a `self.paginate_by = 50` in the `get()` method for the view from when I was testing things.

Reply:
Working now, thanks! +1